runtime/src/bank.rs

```diff
@@ -6385,16 +6373,6 @@ impl Bank {
             ApplyFeatureActivationsCaller::FinishInit,
             debug_do_not_add_builtins,
         );
-
-        if self
-            .feature_set
-            .is_active(&feature_set::cap_accounts_data_len::id())
```

Comment:
same as above. For context, the follow up tasks for #29743 is to introduce a compute_budget instruction to request account data size limit per tx, and charge a fee to economically incentive better space utilization.